build/canary.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+const pkgFile = path.resolve(__dirname, '..', 'package.json');
+const pkg = JSON.parse(fs.readFileSync(pkgFile));
```

Comment:
Should we run standard-version instead? It updates a bunch more stuff, including package-lock, bower.json and the changelog.

Reply:
This repository doesn't use `package-lock.json`, and I wasn't aware Bower was still a thing, haha. 😆 

What would this look like with `standard-version`? I've never used the tool before.

Also, this too would have been nice to discuss in planning rather than during implementation.